cfn-resources/search-index/docs/README.md

```diff
@@ -142,7 +146,7 @@ _Update requires_: [Replacement](https://docs.aws.amazon.com/AWSCloudFormation/l
 
 #### Mappings
 
-_Required_: Yes
+_Required_: No
```

Comment:
is this fixing a bug?

Reply:
the thing is Mappings is required for search indexes but not for vector searches (in fact it can't be passed in vector indexes). similarly for Fields that is mandatory for vector but can't be passed for search.   So at the end it has to be optional and be checked in the server.